src/test/java/org/opentripplanner/routing/algorithm/raptoradapter/router/street/AccessEgressRouterTest.java

```diff
@@ -0,0 +1,228 @@
+package org.opentripplanner.routing.algorithm.raptoradapter.router.street;
+
+import static com.google.common.truth.Truth.assertThat;
+
+import java.time.Duration;
+import java.util.Collection;
+import java.util.Set;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.opentripplanner._support.geometry.CoordinateHelper;
+import org.opentripplanner.framework.geometry.WgsCoordinate;
+import org.opentripplanner.model.GenericLocation;
+import org.opentripplanner.routing.algorithm.GraphRoutingTest;
+import org.opentripplanner.routing.api.request.RouteRequest;
+import org.opentripplanner.routing.api.request.StreetMode;
+import org.opentripplanner.routing.api.request.request.StreetRequest;
+import org.opentripplanner.routing.graph.Graph;
+import org.opentripplanner.routing.graphfinder.NearbyStop;
+import org.opentripplanner.street.model.vertex.TransitStopVertex;
+import org.opentripplanner.street.search.TemporaryVerticesContainer;
+import org.opentripplanner.street.search.state.State;
+import org.opentripplanner.transit.model.framework.FeedScopedId;
+
+class AccessEgressRouterTest extends GraphRoutingTest {
+
+  private Graph graph;
+
+  private TransitStopVertex stop1;
+  private TransitStopVertex stop2;
```

Comment:
Is this a hard rule in otp? I find that shortening names with 1, 2, 3 or A, B, C in test code often improves readability. Since test code is so concrete that descriptive names often turn out to be super long. And long names often leads to ugly formatted code with lots of line breaks.

Reply:
I updated the names to be more descriptive.